src/Sulu/Bundle/AdminBundle/Resources/js/components/Button/tests/__snapshots__/Button.test.js.snap

```diff
@@ -1,129 +1,222 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Should render the button with icon 1`] = `
-<button
-  class="button secondary hasText"
-  type="button"
->
-  <span
-    aria-label="su-plus"
-    class="su-plus buttonIcon"
-  />
-  <span
-    class="buttonText"
+<div>
+  <button
+    class="button secondary hasText"
+    type="button"
   >
-    Add something
-  </span>
-</button>
+    <span
+      aria-label="su-plus"
+      class="su-plus buttonIcon"
+    />
+    <span
+      class="buttonText"
+    >
+      Add something
+    </span>
+  </button>
+</div>
 `;
 
 exports[`Should render with skin icon 1`] = `
-<button
-  class="button icon"
-  type="button"
-/>
+<div>
+  <button
+    class="button icon"
+    type="button"
+  />
+</div>
 `;
 
 exports[`Should render with skin icon and active 1`] = `
-<button
-  class="button icon active"
-  type="button"
-/>
+<div>
+  <button
+    class="button icon active"
+    type="button"
+  />
+</div>
 `;
 
 exports[`Should render with skin icon and dropdown icon 1`] = `
-<button
-  class="button icon"
-  type="button"
->
-  <span
-    aria-label="su-angle-down"
-    class="su-angle-down dropdownIcon"
-  />
-</button>
+<div>
+  <button
+    class="button icon"
+    type="button"
+  >
+    <span
+      aria-label="su-angle-down"
+      class="su-angle-down dropdownIcon"
+    />
+  </button>
+</div>
 `;
 
 exports[`Should render with skin icon and text 1`] = `
-<button
-  class="button icon hasText"
-  type="button"
->
-  <span
-    class="buttonText"
+<div>
+  <button
+    class="button icon hasText"
+    type="button"
   >
-    Icon Text
-  </span>
-</button>
+    <span
+      class="buttonText"
+    >
+      Icon Text
+    </span>
+  </button>
+</div>
 `;
 
 exports[`Should render with skin link 1`] = `
-<button
-  class="button link"
-  type="button"
-/>
+<div>
+  <button
+    class="button link"
+    type="button"
+  />
+</div>
 `;
 
 exports[`Should render with skin link and dropdown icon 1`] = `
-<button
-  class="button link"
-  type="button"
->
-  <span
-    aria-label="su-angle-down"
-    class="su-angle-down dropdownIcon"
-  />
-</button>
+Object {
+  "asFragment": [Function],
+  "baseElement": <body>
+    <div>
+      <button
+        class="button link"
+        type="button"
+      >
+        <span
+          aria-label="su-angle-down"
+          class="su-angle-down dropdownIcon"
+        />
+      </button>
+    </div>
+  </body>,
+  "container": <div>
```

Comment:
This looks strange look like we are creating here a false snapshot. I think the snapshot is done on the false element.

Reply:
You're right. I forgot to migrate the [test you mentioned here](https://github.com/sulu/sulu/pull/6721#discussion_r931481107) which results in printing the whole `render()` return value to the snapshot file.

Reply:
Yeah think moving the snapshot inside the `test` like its done in the other tests should fix it :)